packages/server/src/api/todo/social/getLikedBy.ts

```diff
@@ -26,10 +27,12 @@ export default function (server: Server) {
         .leftJoin(User, 'user', 'like.creator = user.did')
         .leftJoin(ProfileIndex, 'profile', 'profile.creator = user.did')
         .where('like.subject = :uri', { uri })
-        .orderBy('like.createdAt')
+        .orderBy('like.createdAt', 'DESC')
 
       if (before !== undefined) {
-        builder.andWhere('like.createdAt < :before', { before })
+        builder.andWhere('like.createdAt < :before', {
+          before: dateToDb(before),
```

Comment:
Fixes an issue where the database communicates datetimes in the string format `YYYY-MM-DD hh:mm:ss`, but we want to communicate using ISO date strings.

Reply:
hmmm `like.createdAt` should be an ISO timestamp since it's supplied by the client & not by the DB 🤔 

paul & i talked about moving all our dates in the db over to ISO timestamps to avoid confusion like this

Reply:
Ah, in this case we're taking an ISO string from the client and converting it to the date format that the database likes.